chromium_src/chrome/browser/ui/views/frame/browser_view.h

```diff
@@ -19,13 +19,11 @@
 #define BrowserWindow BraveBrowserWindow
 #define BrowserViewLayout BraveBrowserViewLayout
```

Comment:
Looks like this is a duplicate entry with L28 (similar to how #undef was duplicated).